### ISSUE TITLE
dead  link-outs  from the PAINT

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1630]
some of our link-outs  from the PAINT data aren't working. This might be a GO issue, I think @mah11  will know  why but it isn't  at all urgent.

<img width="1006" alt="Screenshot 2020-10-30 at 10 15 38" src="https://user-images.githubusercontent.com/7359272/97751480-4eb3ae80-1aea-11eb-8bd2-26af23f182c2.png">

**Comment by ValWood** [comment]
Seem to link OK in Amigo. I thought we used the same DB source info.

http://amigo.geneontology.org/amigo/gene_product/PomBase:SPAC2F3.15
(filter on evidence IBA)

**Comment by kimrutherford** [comment]
Which gene page is that and which links aren't working?

**Comment by ValWood** [comment]
SPAC2F3.15

**Comment by ValWood** [comment]
no actually, pef1 is an example

**Comment by kimrutherford** [comment]
I checked all the links on the pef1 page.   I get errors for the RGD links and this link didn't load: http://www.pantree.org/node/annotationNode.jsp?id=PTN000623091

**Comment by ValWood** [comment]
some of the links don't resolve
ZDB-GENE-010131-2 
locus:2011761
CAL0000191263
and some it isn't clear which database (I. the RGD ones)

I'll report all of this to GO on Monday.....

**Comment by kimrutherford** [comment]
> some of the links don't resolve

Ah, right.  Got it now - some IDs aren't links.  That's probably a configuration issue at our end.  I'll check.

**Comment by ValWood** [comment]
OK let me know if not. I'tt report the remaining to GO.

**Comment by kimrutherford** [comment]
All the IDs in your screenshot are now links.

**Comment by ValWood** [comment]
OK great. so just the rat IDs that don't resolve, I will report that

**Comment by kimrutherford** [comment]
Can you give me an example so I can fix the configuration?

**Comment by ValWood** [comment]
Actually, before I submit to GO tracker
 just checking this isn't a config issue and we aren't missing a prefix?

----

RGD IDs not resolving

see for example
https://www.pombase.org/gene/SPCC16C4.11

+ | cyclin-dependent protein serine/threonine kinase activity
-- | --

(Show details)


cyclin-dependent protein serine/threonine kinase activity | IBA with 70486 , MGI:104772 , FBgn0019949 , 2319 , Q00534 , ZDB-GENE-081022-110 , A0A1D8PDA6 , mcs6 , S000005963 , S000005952 , S000000364 , CAL0000191263 , 621120 , DDB_G0288677 , FBgn0004106 , FBgn0263237 , P61075 , MGI:88357 , S000006365 , WBGene00000405 , cdk9 , PTN000623091 , lsk1 , Q8IJQ1 , 70514 , MGI:88351 , P06493 , S000001622 , P50750 , 621124 , C9K505 , P24941 , DDB_G0272813 , FBgn0005640 , srb10 , locus:2011761 , ZDB-GENE-010131-2 , Q00646 , P11802 , FBgn0013762 , O94921 , cdc2
-- | --


The RGS IDs are the ones which are just numbers like 621124

**Comment by kimrutherford** [comment]
> Actually, before I submit to GO tracker
> just checking this isn't a config issue and we aren't missing a prefix?

I think the url_syntax field for RGD in GO.xrf_abbs is wrong.  The url_example works but we use the url_syntax for the links. 

Here's what the entry looks like at the moment:

```
abbreviation: RGD
database: Rat Genome Database
synonym: RAD
synonym: RGDID
generic_url: https://rgd.mcw.edu/
local_id_syntax: [0-9]{4,7}
url_syntax: https://rgd.mcw.edu/rgdweb/search/search.html?[example_id]
example_id: RGD:2004
url_example: https://rgd.mcw.edu/rgdweb/search/search.html?term=2004
entity_type: SO:0000704 ! gene
```

 It looks like it should be:
``` 
url_syntax: https://rgd.mcw.edu/rgdweb/search/search.html?term=[example_id]`
```

**Comment by kimrutherford** [comment]
Report by Val: https://github.com/geneontology/go-site/issues/1570

**Comment by kimrutherford** [comment]
Fixed by #1634
